src/sentry/tasks/store.py

```diff
@@ -677,4 +695,4 @@ def save_event_highcpu(
     project_id: int | None = None,
     **kwargs: Any,
 ) -> None:
-    _do_save_event(cache_key, data, start_time, event_id, project_id, **kwargs)
+    _do_save_event(ConsumerType.Events, cache_key, data, start_time, event_id, project_id, **kwargs)
```

Comment:
This task might be defunct. I don't see any callsites for it, but it looks like the cleanup for this task wasn't completed.

Reply:
I asked arpad about it, but I also think its safe to remove.